docs/user-guide/widgets.md

```diff
@@ -345,7 +347,17 @@ Adding a table widget to the map, a panel like the following opens:
 
 <img src="../img/widgets/table-panel.jpg" class="ms-docimage"  style="max-width:450px;"/>
 
-The toolbar on the top of this panel is similar to the one present in [Chart section](#chart). Here the user is allowed to:
+From the top toolbar of this panel the user is allowed to:
```

Comment:
@ElenaGallo if you want to specify this at table widget level and chart level why you shouldn't do that also for counter widgets?